pkg/sysinfo/sysinfo.go

```diff
@@ -3,23 +3,42 @@ package sysinfo
 // SysInfo stores information about which features a kernel supports.
 // TODO Windows: Factor out platform specific capabilities.
 type SysInfo struct {
+	// Whether support APpArmor or not
```

Comment:
For ``SysInfo`` and ``cgroup*Info``, maybe you should add something like : "Wether the current platform supports AppArmor" (or s/platform/kernel/).